src/classes/RD2_OpportunityEvaluationService_TEST.cls

```diff
@@ -1863,6 +1863,34 @@ private class RD2_OpportunityEvaluationService_TEST {
         
     }
 
+    /**
+    * @description Verifies that when Disable First Installment is true,
+    *   No installment will be created on RD insert regardless of NPSP Installment Auto Creation Setting
+    */
+    @IsTest
+    private static void shouldNotCreateFirstInstallmentWhenDisableFirstInstallmentIsTrue() {
```

Comment:
Can you please identify which existing tests map to AC1, AC2 and AC3? If they do not map, let's create a unit test for them.

Reply:
[AC2](https://github.com/SalesforceFoundation/NPSP/blob/c24c3318768f8a1df3b1e714fba26f6bbba4a21c/src/classes/RD2_OpportunityEvaluationService_TEST.cls#L425), [AC3](https://github.com/SalesforceFoundation/NPSP/blob/c24c3318768f8a1df3b1e714fba26f6bbba4a21c/src/classes/RD2_OpportunityEvaluationService_TEST.cls#L343), and [AC4](https://github.com/SalesforceFoundation/NPSP/blob/c24c3318768f8a1df3b1e714fba26f6bbba4a21c/src/classes/RD2_OpportunityEvaluationService_TEST.cls#L367) is covered with NPSP Auto Creation Setting Tests.